kiwi/command_process.py

```diff
@@ -33,10 +33,13 @@
 
 class CommandProcess(object):
     """
-    Implements processing of non blocking Command calls
+    **Implements processing of non blocking Command calls**
 
     Provides methods to iterate over non blocking instances of
     the Command class with and without progress information
+
+    :param subprocess command: instance of subprocess
+    :param string log_topic: topic string for logging
```

Comment:
Not sure if you want to have parameters at the class docstring. It's probably a matter of taste if you add them on the constructor or on the class. The only issue is that it should be consistent. :wink:

Reply:
yes it was a matter of taste and we did it in the class docstring all over the place and it is rendered fine in sphinx. Thus I kept that concept and think it's ok if reading the class intro also describes how to construct on object of that class. I think we don't have a consistency problem because we did that from the beginning on, so there should not be any __init__ constructor with a docstring